docs/ARCHITECTURE.md

```diff
@@ -328,8 +328,7 @@ npm install
 npm run test
 ```
 
-Currently v8.0.0 <= Node.JS <= v11.15.0 is required because web3 v1.0.0-beta.54 requires it. In the future this restriction will be lifted.
```

Comment:
why did you remove this?

Reply:
Because of [this PR](https://github.com/enigmampc/enigma-p2p/pull/220)